### PR TITLE
Release notes for 0.19

### PR DESCRIPTION
**Pull request by pitrou**

### CODE REVIEW

File: CHANGE_LOG [hunk 1]
```diff
@@ -1,5 +1,66 @@
+
+Version 0.19.0
+--------------
+
+This version introduces memory management in the Numba runtime, allowing to
+allocate new arrays inside Numba-compiled functions.  There is also a rework
+of the ufunc infrastructure, and an optimization pass to collapse cascading
+array operations into a single efficient loop.
+
```
Comment (seibert) [inline CHANGE_LOG]: Let's also add a two additional notes:

```
.. warning:: 
Support for Windows XP and Vista with all compiler targets and support for 32-bit platforms (Win/Mac/Linux) 
with the CUDA compiler target are deprecated.  In the next release of Numba, the oldest version of Windows
supported will be Windows 7.  Numba will continue to support CPU compilation on Linux and Windows 32-bit
platforms.

Known issues:

* There are some performance regressions in very short running nopython functions due to the additional overhead
incurred by memory management.  We will work to reduce this overhead in future releases.

**Comment by pitrou** [comment]
Good point, I'll add those.

**Comment by pitrou** [comment]
Added.

**Comment by seibert** [comment]
OK, we'll hold this PR open until just before release in case any addition PRs need to be merged.

**Pull request merged**